prow/plugins/label/label.go

```diff
@@ -43,15 +43,23 @@ type assignEvent struct {
 var (
 	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind|sig)\s*(.*)$`)
 	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|priority|kind|sig)\s*(.*)$`)
+	statusRegex             = regexp.MustCompile(`(?m)^/status\s+(.+)$`)
 	sigMatcher              = regexp.MustCompile(`(?m)@kubernetes/sig-([\w-]*)-(misc|test-failures|bugs|feature-requests|proposals|pr-reviews|api-reviews)`)
-	chatBack                = "Reiterating the mentions to trigger a notification: \n%v."
-	nonExistentLabelOnIssue = "Those labels are not set on the issue: `%v`."
-	kindMap                 = map[string]string{
+	chatBack                = "Reiterating the mentions to trigger a notification: \n%v"
+	nonExistentLabelOnIssue = "Those labels are not set on the issue: `%v`"
+	mustBeSigLead           = "You must be a sig lead to add the status label"
+	statusMap               = map[string]string{
+		"approved-for-milestone": "status/approved-for-milestone",
+		"in-progress":            "status/in-progress",
+		"in-review":              "status/in-review",
+	}
+	kindMap = map[string]string{
 		"bugs":             "kind/bug",
 		"feature-requests": "kind/feature",
 		"api-reviews":      "kind/api-change",
 		"proposals":        "kind/design",
 	}
+	MILESTONE_MAINTAINERS_ID = 2460384
```

Comment:
ok to hardcode this?  or should I hardcore the github team name (kubernetes-milestone-maitainers and then use ListTeams() and then get the id from there).